<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InserimentoTessera</class>
 <widget class="QWidget" name="InserimentoTessera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cliente ID: </string>
   </property>
  </widget>
  <widget class="QLabel" name="l_nome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_cognome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cognome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_indirizzo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Indirizzo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_telefono">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefono:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_email">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Codice Tessera:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_codice">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="line_data">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data di iscrizione:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero di punti:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_punti">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_inserisci">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
